Cargo.toml

```diff
@@ -42,5 +42,11 @@ debug = false
 debug-assertions = false
 
 
+[workspace.dependencies]
+cw-multi-test = { version = "0.16.0" }
```

Comment:
You shouldn't need cw-multi-test, we re-export the types that you should need in boot-core

Reply:
Fixed by [2f191b4](https://github.com/AbstractSDK/cw-plus/pull/1/commits/2f191b487f00b14b92c6e7d9e62efdd4fe7293ea) and [f30c80a](https://github.com/AbstractSDK/BOOT/pull/77/commits/f30c80a279cdabe08b97de31be78703029aa01a3)